aws/resource_aws_globalaccelerator_endpoint_group_test.go

```diff
@@ -161,6 +195,112 @@ resource "aws_globalaccelerator_endpoint_group" "example" {
 `, rInt)
 }
 
+func testAccGlobalAcceleratorEndpointGroup_alb_clientip(rInt int) string {
+	return fmt.Sprintf(`
+resource "aws_lb" "lb_test" {
+  name            = "%d"
+  internal        = false
+  security_groups = ["${aws_security_group.alb_test.id}"]
+  subnets         = ["${aws_subnet.alb_test.*.id[0]}", "${aws_subnet.alb_test.*.id[1]}"]
+
+  idle_timeout               = 30
+  enable_deletion_protection = false
+
+  tags = {
+    Name = "TestAccAWSALB_basic"
+  }
+}
+
+variable "subnets" {
+  default = ["10.0.1.0/24", "10.0.2.0/24"]
+  type    = "list"
+}
+
+data "aws_availability_zones" "available" {}
```

Comment:
Please rebase or merge to the current `HEAD` and change to

```hcl
data "aws_availability_zones" "available" {
  state = "available"

  filter {
    name   = "opt-in-status"
    values = ["opt-in-not-required"]
  }
}
```

otherwise the acceptance tests may fail with attempting to use any enabled Local Zone in `us-west-2`.